### ISSUE TITLE
Add option to specify folder where assets should be copied to (instead of textfile dir)

### ISSUE THREAD

**Issue by gsantner** [gsantner/markor#668]
#### General information

* **App version:** v2.0
* **System:** Android 9

**Comment by gsantner** [comment]
Feedback by @Agrotsa from [project website comment](https://github.com/gsantner/gitalk-gsantner-net/issues/7#issuecomment-526933732)

----

Nevertheless, I have a small criticism to do, I hope you find it constructive.
Seems that markor copies the atached media to the folder where is the note in question. This may end being a mess if you have a lot of notes and media files in said folder since all them are stored there together no matter to which noted they belong. It wouldnt be really a problem if one only has local notes, the problem comes when one wants to share notes with media inserted.
Id like to suggest to consider the method used by Typora (or any other similar, I just know of Typora doing this, thats why I'm mentioning it), which is almost the same that web browsers use when you save a web page: creating a new subfolder for each document that contains atachments, with the name of the page/note and then store right there all the media files **corresponding to that single document**. Firefox creates a folder called NameOfTheHTMLFile_files, Typora creates TitleOfTheNote.assets, but in esence is the same, the extension name doesnt have much importance.
For example, if I'm using Typora to write a note titled Vacations_itinerary.md and want to insert some photos of places that I've visited in my vacations trip I would end with a .md file and a .assets folder with **only** the images that I inserted in said MD file; and I could easily upload/sinchronize both the .md and the .assets folder to my cloud and share it to people or open it in my PC or tablet when I'm at home, etc.
I think this is more efficient than having a folder with like 35 notes and, lets say, 150 images. Sure I could upload the MD file to my cloud and search, select and upload by hand the images atached but is very time consuming and probably most users would end getting tired of the process.

----
